src/api/sizeme-api.js

```diff
@@ -363,16 +363,33 @@ function doMatch (fitRequest, token, useProfile) {
 function getRecommendedFit (fitResults, optimalFit) {
     const optFit = optimalFit ? optimalFit : DEFAULT_OPTIMAL_FIT;
     const maxDist = uiOptions.maxRecommendationDistance || 9999;
-    const [bestMatch] = fitResults
-        .filter(([, res]) => res.totalFit >= 1000 && res.accuracy > 0)
-        .reduce(([accSize, fit], [size, res]) => {
-            const newFit = Math.abs(res.totalFit - optFit);
-            if (newFit <= maxDist && (!accSize || newFit < fit)) {
-                return [size, newFit];
-            } else {
-                return [accSize, fit];
-            }
-        }, [null, 0]);
+    let [bestMatch] = [];
```

Comment:
Tämä on sama kuin `let bestMatch = undefined`. Ehkä siistimpää olisi esitellä bestMatch kummassakin if-haarassa erikseen: `const [bestMatch] = fitResults..` ja palauttaa myös if-haarojen sisältä suoraan.

Reply:
Joo, tosiaan jos se palautuskin tapahtuu haaroista suoraan (heh), niin on varmaan ok.  Siellä joku automaattinen tarkistus nimittäin valitti kaikista muista mun mielestä siisteistä vaihtoehdoista.